service/src/main/java/bio/terra/tanagra/app/AuthInterceptor.java

```diff
@@ -97,6 +97,9 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         userAuth =
             new UserAuthentication(
                 userId, bearerToken.getToken(), UserAuthentication.TokenType.BEARER_TOKEN);
+      } else if (authConfiguration.isDisableChecks()) {
+        LOGGER.warn("Authentication checks are disabled");
```

Comment:
optional

Authentication checks are disabled. This should only happen for local development.

Reply:
done.